pages/overview.js

```diff
@@ -1,3 +1,70 @@
-export default function OverviewPage() {
-  return <h1>Overview page</h1>;
+import { StyledSection } from "./locationdetail/[id]";
+import { StyledLink } from "@/components/LocationList";
+import styled from "styled-components";
+
+const StyledOverviewCard = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px grey solid;
+  padding: 1rem;
+  margin-bottom: 1rem;
+  width: 18rem;
+
+  & > div {
+    display: flex;
+    flex-direction: row;
+  }
+`;
+
+const StyledOverviewColony = styled.article`
+  padding: 1rem;
+  border: 1px solid grey;
+  margin: 0.3rem;
+`;
+
+const StyledColonySection = styled.section`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+`;
+
+const StyledOverviewLocation = styled.p`
+  font-weight: bold;
+`;
+
+export default function OverviewPage({ locations, colonies }) {
+  return (
+    <>
```

Comment:
We don't need the React Fragment if we are only returning one element from the component (in this case we are only returning `StyledSection`).